package/base-files/files/sbin/sysupgrade

```diff
@@ -157,15 +166,15 @@ add_overlayfiles() {
 		local conffiles=$1.conffiles
 		local keepfiles=$1.keepfiles
 
-		list_conffiles | cut -f2 -d ' ' | sort -u > "$conffiles"
+		list_conffiles | cut -f3- -d ' ' | sort -u > "$conffiles"
```

Comment:
Just adapts to the new list_conffiles format (<hash><space><space><filename with spaces>)